src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -387,7 +390,27 @@ private FcrepoResponse importContainer(final URI uri, final Model model) throws
             logger.debug("Using Bagit checksum ({}) for file ({})", checksum, containerFile.getPath());
             builder = builder.digest(checksum);
         }
-        return builder.preferLenient().perform();
+        final FcrepoResponse response = builder.preferLenient().perform();
+        if (response.getStatusCode() == 410 && config.overwriteTombstones()) {
+            deleteTombstone(response);
+            return builder.preferLenient().perform();
```

Comment:
I am a little surprised that calling the `builder` twice works... particularly given the inputstream (https://github.com/fcrepo4-labs/fcrepo-import-export/pull/76/files#diff-f6498b3791251ba98aa82e1e70ebbe51R384).

It looks like the integration test is working with an empty body: https://github.com/fcrepo4-labs/fcrepo-import-export/pull/76/files#diff-d39f7f254dc82ffc7b294a4db96d2c5eR331